src/pd/metrics.rs

```diff
@@ -14,24 +14,17 @@
 use prometheus::*;
 
 lazy_static! {
-    pub static ref PD_SEND_MSG_HISTOGRAM: Histogram =
-        register_histogram!(
-            "tikv_pd_msg_send_duration_seconds",
-            "Bucketed histogram of PD message send duration",
-             exponential_buckets(0.0005, 10.0, 7).unwrap()
```

Comment:
Why not keep original buckets?

Reply:
It seems not very useful, and can't distinguish the message type.